l5kit/l5kit/cle/closed_loop_evaluator.py

```diff
@@ -0,0 +1,199 @@
+from typing import Dict, Iterable, List, Optional
+
+import torch
+
+from l5kit.cle.composite_metrics import SupportsCompositeMetricCompute
+from l5kit.cle.metrics import SupportsMetricCompute
+from l5kit.cle.validators import SupportsMetricValidate, ValidatorOutput
+from l5kit.simulation.unroll import SimulationOutput
+
+
+class EvaluationPlan:
```

Comment:
just to check: this doesn't have a method to:
- evaluate metrics
- evaluate validators
- evaluate composed metrics
 right?

that will come later somewhere else

Reply:
Correct, that will come from the `ClosedLoopEvaluator`